SAKPaaS/src/app/components/map/buttons.scss

```diff
@@ -24,3 +24,46 @@ button.ol-rotate-reset:hover {
   background-color: var(--silver);
   cursor: pointer;
 }
+
+div.ol-overlaycontainer-stopevent {
+  z-index: auto !important;
+}
+
+div.ol-zoom {
+  top: auto;
+  left: auto;
+  bottom: 40px;
+  right: 30px;
+  background-color: transparent;
+}
+
+div.ol-attribution a {
+  color: var(--greenblue);
+}
+
+div.ol-rotate {
```

Comment:
sieht man nicht. Zumindestens auf meinem Laptop 🤷‍♂️ 😄 
![image](https://user-images.githubusercontent.com/52164046/81157122-f2cafb80-8f86-11ea-966b-70c7cf115cb6.png)

Reply:
Öhm, den blendet OL nur ein, wenn du die Karte gedreht hast, also CMD-Alt-Maus.

Reply:
Ja I know, aber die is ja grad gedreht. Der Button wird aber von der usercard überdeckt (du siehst ja ganz leicht den Anfang auf dem Bild) ^^

Reply:
Design Flaw, I guess @cgomm xD
Wo soll er denn alternativ hin? Daneben? Die Höhe der User Card ist halt auch dynamisch, deswegen würde mir jetzt keine einfache Möglichkeit einfallen, das optional zu triggern.

Reply:
@nclskfm @timgrohmann wieso wird denn die Email da mit angezeigt? Ich würde die da nicht mit anzeigen.. Würde das das Problem lösen?

Reply:
Guuter Fix xD Dann würde ich das erstmal so lassen, oder? Die Usercard wird doch eh gerade angepasst, ne? @nclskfm

Reply:
allet klar xD